### PR TITLE
Remove unnecessary interrupt call

### PR DESCRIPTION
**Pull request by tajila**
Remove unnecessary interrupt call

Signed-off-by: Tobi Ajila <atobia@ca.ibm.com>

### CODE REVIEW

**Comment by tajila** [comment]
jenkins test sanity zlinux jdknext

File: src/java.base/share/classes/java/lang/Thread.java [hunk 1]
```diff
@@ -1693,22 +1693,20 @@ public final void stop() {
...
-                    interrupted = true;
-                    interrupt0();  // inform VM of interrupt
-                    b.interrupt(this);
-                    return;
-                }
-            }
+            Interruptible b = nioBlocker;
+            if (b != null) {
+                b.interrupt(this);
+                }
```
Comment (JasonFengJ9) [inline src/java.base/share/classes/java/lang/Thread.java:1708]: There are a few extra spaces at end of this line, and the indentation doesn't match.

**Comment by JasonFengJ9** [comment]
jenkins test sanity osx jdknext

**Comment by JasonFengJ9** [comment]
> 16:47:32  Exception in thread "main" java.lang.UnsupportedOperationException
16:47:32  	at java.base/java.lang.Thread.stop(Thread.java:1650)
16:47:32  	at j9vm.test.clinit.NewDuringClinit.test(NewDuringClinit.java:108)
16:47:32  	at j9vm.test.clinit.BlockDuringClinitTest.run(BlockDuringClinitTest.java:30)
16:47:32  	at j9vm.test.clinit.BlockDuringClinitTest.main(BlockDuringClinitTest.java:44)
01:58:38  Cancelling nested steps due to timeout

Known failure (https://github.com/eclipse-openj9/openj9/issues/16028)

**Review by JasonFengJ9 — APPROVED** [review]
LGTM

**Pull request merged**